<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellmlModelRepositoryWindow</class>
 <widget class="QDockWidget" name="CellmlModelRepositoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CellML Model Repository</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="subLayout">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="filterLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterValue"/>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="toolTip">
         <string>Refresh</string>
        </property>
        <property name="statusTip">
         <string>Refresh the list of CellML models</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/CellMLModelRepository.qrc">
          <normaloff>:/oxygen/actions/view-refresh.png</normaloff>:/oxygen/actions/view-refresh.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../../../misc/Core/res/Core.qrc">
     <normaloff>:/oxygen/actions/edit-copy.png</normaloff>:/oxygen/actions/edit-copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy the selection to the clipboard</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../misc/Core/res/Core.qrc"/>
  <include location="../res/CellMLModelRepository.qrc"/>
 </resources>
 <connections/>
</ui>
